spec/source_finder/source_file_globber_ruby_spec.rb

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+require 'spec_helper'
+require 'source_finder/source_file_globber'
+
+describe SourceFinder::SourceFileGlobber do
+  let_double :globber
+
+  subject(:source_file_globber) do
+    described_class.new(globber: globber)
+  end
+
+  describe '#ruby_dirs_arr' do
+    context 'when unconfigured' do
+      subject { source_file_globber.ruby_dirs_arr }
+      it { is_expected.to eq(%w(app config db feature lib spec src test)) }
+    end
+    let_double :ruby_dirs
+    context 'when configured' do
+      before { source_file_globber.ruby_dirs_arr = ruby_dirs }
+      subject { source_file_globber.ruby_dirs_arr }
+      it { is_expected.to eq(ruby_dirs) }
+    end
+  end
+
+  describe '#ruby_files_glob' do
+    context 'with everything unconfigured' do
+      expected_glob = '{Rakefile,{*,.*}.{gemspec,rake,rb},{app,config,' \
+                      'db,feature,lib,spec,src,test}/**/{*,.*}.' \
+                      '{gemspec,rake,rb}}'
+      subject { source_file_globber.ruby_files_glob }
+      it { is_expected.to eq(expected_glob) }
+    end
+  end
+
+  describe '#ruby_files_arr' do
+    before do
+      expect(globber).to receive(:glob)
+        .with('{Rakefile,{*,.*}.{gemspec,rake,rb},' \
+              '{app,config,db,feature,lib,spec,src,test}/**/{*,.*}.' \
+              '{gemspec,rake,rb}}')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.